pyansys_sphinx_theme/static/css/theme.css

```diff
@@ -179,3 +179,180 @@ p.rubric {
     margin-top: 0.75em;
     margin-bottom: 0.75em;
 }
+
+
+
+/*
+###############
+   table 
+###############
+*/
+
+table {
+  width:100%;
+  max-width: 100%; 
+  border-spacing: 0;
+  border-collapse: collapse;
+  overflow: hidden;
+  vertical-align: middle;
+
+  /* Disabling scroll bars */
+  overflow-y: scroll;
+  scrollbar-width: none; /* Firefox */
+  -ms-overflow-style: none;  /* Internet Explorer 10+ */
+  }
+
+tr { 
+  background-color: white;
+}
+
+th {
+  background-color: rgb(255, 183, 27, 0.55);
+}
+
+tr:nth-child(odd){ 
+  background-color: white;
```

Comment:
I don't know if you are seeing this as well, but this declaration is getting overridden by something within the html itself. If that's the case, I think the only way to get around it is to add `!important`, something like:

`background-color: white !important;`

!important can cause problems, because it overrides the natural organization and structure of CSS. Sometimes though, there is no alternative...

Reply:
mmhh it is interesting that for the ``_autosummary`` definition, the white works.

We could use important for the moment, until we see any trouble or find a better solution.

Reply:
I'm checking again... I think it is fine with the light gray (it is very light), so I think we can leave it as it is and move on. It shows a bit but if you believe we are going to have side effects, I'm fine with that gray around.

Reply:
i'm not able to get the white color in the odd rows of the dataframe, even if i use ``!important``. I think we can leave it like that for the moment. 

I guess this is going to require review the whole CSS or something else which I have not much expertise.

Reply:
Yea, let's not go too far. The best CSS is the CSS that works!